### DISCUSSION TITLE
Planned use of AI for @modelcontextprotocol on GitHub

### DISCUSSION THREAD

**Discussion by jspahrsummers** [modelcontextprotocol/.github#95] [Announcements]
Hi folks! Just a quick announcement that we're planning to experiment with AI helping us manage incoming issues, pull requests, and discussions for the @modelcontextprotocol org on GitHub.

While we put this together:
1. Maintainers' participation on issues, PRs, and discussions may come more slowly
2. You may get AI responses that are low usefulness, or get automatic notifications about AI responses that we then manually delete

Please bear with us! Our hope is to build something awesome here that helps us scale with the community, while also creating an even better, more responsive experience for contributors. 🙏

**Comment by adamavenir** [comment]
@jspahrsummers any chance this will be built as open source? would be super cool to see MCP use MCP to build MCP

**Reply by jspahrsummers** [reply]
Not sure what we are or aren't building yet, but would love to open source it if possible! 😄

**Reply by adamavenir** [reply]
@jspahrsummers maybe it could be planned in public, too? :)

**Reply by Jacck** [reply]
we should keep human around

**Comment by coltonbatts** [comment]
![Image 1-1-25 at 4 24 PM](https://github.com/user-attachments/assets/8ab2cfbf-aaa0-4dbe-af43-69d98996efd8)
im trying to make number go up.
